drivers/serial/uart_native_posix.c

```diff
@@ -379,7 +401,12 @@ static void np_add_uart_options(void)
 		(void *)&auto_attach_cmd, NULL,
 		"Command used to automatically attach to the terminal, by "
 		"default: '" CONFIG_NATIVE_UART_AUTOATTACH_DEFAULT_CMD "'"},
-
+		IF_ENABLED(CONFIG_UART_NATIVE_WAIT_PTS_READY_ENABLE, (
+			{false, false, true,
+			"pts-wait", "", 'b',
+			(void *)&wait_pts, NULL,
+			"Wait for pts reader to become ready"},)
```

Comment:
From this option name and description, I think users may have quite some trouble guessing what it does.
Would it make sense to call it `uart-wait`, and describe it as: `"hold writes to the uart/pts until a client is connected/ready"`?

Reply:
Just to align with other already used cmd line options, what about `--wait_uart`?

Reply:
As you prefer. (Somehow I'm not too happy that I named the other with uart at the end as it is a bit more difficult for users to realize they are all uart related)

Reply:
Yep, I'd like to rename all of them to `--uart_xxx` but this sadly would broke backward compatability :-(

Reply:
It would. There is the option of adding aliases for backwards compatibility though.

Reply:
If it is possible, let's do it in another PR.